tests/lib/rules/no-assert-ok.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @fileoverview Forbid the use of assert.ok/notOk and suggest other assertions.
+ * @author Kevin Partington
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/no-assert-ok"),
+    RuleTester = require("eslint").RuleTester;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
+
+ruleTester.run("no-assert-ok", rule, {
+    valid: [
+        "QUnit.test('Name', function (assert) { assert.strictEqual(a, b); });",
+        "QUnit.test('Name', function (assert) { assert.deepEqual(a, b); });",
+        "QUnit.test('Name', function (assert) { assert.propEqual(a, b); });",
+        "QUnit.test('Name', function (foo) { foo.strictEqual(a, b); });",
+        "QUnit.test('Name', function (foo) { foo.deepEqual(a, b); });",
+        "QUnit.test('Name', function (foo) { foo.propEqual(a, b); });",
+        "QUnit.test('Name', function (assert) { strictEqual(a, b); });",
+        "QUnit.test('Name', function (assert) { deepEqual(a, b); });",
+        "QUnit.test('Name', function (assert) { propEqual(a, b); });",
+        "QUnit.test('Name', function () { strictEqual(a, b); });",
+        "QUnit.test('Name', function () { deepEqual(a, b); });",
+        "QUnit.test('Name', function () { propEqual(a, b); });",
+
+        // equal is not within test context
+        "equal(a, b);"
+    ],
+
+    invalid: [
+        {
+            code: "QUnit.test('Name', function (assert) { assert.ok(a); });",
+            errors: [{
+                messageId: "unexpectedLocalOkNotOk",
+                data: { assertVar: "assert",
```

Comment:
Could you please move the `assertVar` property key and value to the next line, indented by one level relative to the start of this line? (Apologies, I will need to figure out why ESLint isn't enforcing this.)

Same for the rest of the `invalid` tests.